core/integration/module_test.go

```diff
@@ -4277,7 +4277,7 @@ class Test {
 				require.NoError(t, c.Close())
 
 				t.Log(logs.String())
-				require.Contains(t, logs.String(), "too bad")
+				require.Regexp(t, "too bad: so sad", logs.String())
```

Comment:
Yikes at the false positive, that's extremely subtle but makes sense...

Were the times this test failed due to one of the OTEL flushing issues and fixed now? Just confirming because my other refactoring PR seems to hit failures on it even more often than before, but not sure if it's likely to be just a timing difference and fixed once I rebase on this PR.

Reply:
I think it's actually the switch to the new plain output that introduced the flakiness - see https://github.com/dagger/dagger/pull/7452#discussion_r1628304802

Definitely timing related, should be fixed after rebasing on this since it applies the aforementioned fix to the plain frontend.

Reply:
Thanks, just did a rebase on this PR and can confirm it passes consistently in the other PR now 🎉